kolibri/core/auth/management/commands/bulkimportusers.py

```diff
@@ -69,17 +70,26 @@
 INVALID_UUID = 12
 
 MESSAGES = {
-    UNEXPECTED_EXCEPTION: _("Unexpected exception [{}]: {}"),
-    TOO_LONG: _("'{}' is too long"),
-    INVALID: _("Not a valid '{}'"),
-    DUPLICATED_USERNAME: _("Duplicated Username"),
+    UNEXPECTED_EXCEPTION: pgettext_lazy(
+        "Error message that might appear when there's a programming error importing a CSV file",
+        "Unexpected error [{}]: {}",
+    ),
+    TOO_LONG: _("Value in column '{}' is too many characters"),
+    INVALID: _("Invalid value in column '{}'"),
+    DUPLICATED_USERNAME: _("Username is duplicated"),
     INVALID_USERNAME: _(
         "Username only can contain characters, numbers and underscores"
     ),
-    REQUIRED_COLUMN: _("The column '{}' is required"),
-    INVALID_HEADER: _("Mix of valid and/or invalid header labels found in first row"),
+    REQUIRED_COLUMN: pgettext_lazy(
+        "Importing a CSV file, there's a column that has not been added to the file",
```

Comment:
`Error message indicating that the CSV file selected for import is missing a required column.`